views/peeps/new.erb

```diff
@@ -0,0 +1,4 @@
+<form action='/peeps' method='post'>
+  <input type='text' name='content' />
+  <input type='submit' value='Post' />
+</form>
```

Comment:
Really good concise and clear method used the whole way through. Very easy to follow and understand.